easybuild/easyconfigs/h/HTSlib/HTSlib-1.3.1-goolf-1.7.20.eb

```diff
@@ -0,0 +1,27 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics 
+
+easyblock = 'ConfigureMake'
+
+name = 'HTSlib'
+version = '1.3.1'
+
+homepage = 'http://www.htslib.org/'
+description = """A C library for reading/writing high-throughput sequencing data.
+ HTSlib also provides the bgzip, htsfile, and tabix utilities"""
+
+toolchain = {'name': 'goolf', 'version': '1.7.20'}
+
+source_urls = ['https://github.com/samtools/htslib/releases/download/%(version)s/']
+sources = [SOURCELOWER_TAR_BZ2]
+
+dependencies = [('zlib', '1.2.8')]
+
+sanity_check_paths = {
+    'files': ["bin/bgzip", "bin/tabix", "bin/htsfile", "lib/libhts.so", "include/htslib/hts.h"],
```

Comment:
@pescobar please use `SHLIB_EXT` rather than hardcoding `.so` & sort these alphabetically while you're at it :)